Update documentation on user storage provider in Quarkus (#19898)
keycloak/keycloak: docs/documentation/server_development/topics/user-storage/configuration.adoc

    }
----

> In the `validateConfiguration()` method we get the configuration variable from the `ComponentModel` and we check to see if that file exists on disk.

Reviewer: ```suggestion
The `validateConfiguration()` method provides the configuration variable from the `ComponentModel` to verify if that file exists on disk.
```

> Notice that we use the `org.keycloak.common.util.EnvUtil.replace()` method. With this method any string that has `${}` within it will replace that with a system property value.
> The `${jboss.server.config.dir}` string corresponds to the `conf/` directory of our server and is really useful for this example.

Reviewer: ```suggestion
Notice that the use of the `org.keycloak.common.util.EnvUtil.replace()` method. With this method any string that includes`${}` will replace that value with a system property value.
The `${jboss.server.config.dir}` string corresponds to the `conf/` directory of our server and is really useful for this example.
```
